File: Programmers/src/JadenCase문자열만들기.java
//import java.util.*;
//
//class Solution {
//    public String solution(String s) {
//        String answer = "";
//        
//        String s2 = s.toLowerCase();
//        char last = ' ';
//        
//        for(char c : s2.toCharArray()){
//            if(last == ' ') c = Character.toUpperCase(c);
//            answer += c;
//            last = c;
//        }
//        
//        return answer;
//    }
//}

class Solution {
    public String solution(String s) {
        String answer = "";
        String[] arr = s.toLowerCase().split("");
        StringBuilder sb = new StringBuilder();
        
        boolean blank = true;
        
        for(String i : arr){
            answer += blank ? i.toUpperCase() : i;
            blank = i.equals(" ") ? true : false;
        }
        
        return answer;
    }
}

File: Programmers/src/Javascript/Lv2/JadenCase문자열만들기.js
function solution(s) {
    var answer = [];
    
    let array = s.split(" ");
    
    for(let i=0; i<array.length; i++) {
        answer.push(array[i].substr(0,1).toUpperCase() + array[i].substr(1,array[i].length).toLowerCase());
    }
    
    return answer.join(" ");
}